configs/CSC-Lumi-HPECray-zen3-MI250X-Slingshot/auxiliary_software_files/compilers.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright 2023 Lawrence Livermore National Security, LLC and other
+# Benchpark Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+compilers:
+- compiler:
+    spec: cce@16.0.0-rocm5.5.1
+    paths:
+      cc:  /opt/cray/pe/cce/16.0.0/bin/craycc
+      cxx:  /opt/cray/pe/cce/16.0.0/bin/crayCC
+      f77:  /opt/cray/pe/cce/16.0.0/bin/crayftn
+      fc:  /opt/cray/pe/cce/16.0.0/bin/crayftn
+    flags:
+      cflags: -g -O2
+      cxxflags: -g -O2 -std=c++14
```

Comment:
I may have missed this in other configs: but I'd recommend against putting C++ standard flags in as a universal choice: each package should generally decide on its own whether it needs to enable a given c++ standard (and many Spack packages do this, which could conflict with this setting).

I also think `-g -O2` should be omitted here, but that should generally not be an error.